published-namespaces.txt

```diff
@@ -320,6 +320,7 @@ Google.Android.Material.Expandable
 Google.Android.Material.FloatingActionButton
 Google.Android.Material.ImageView
 Google.Android.Material.Internal
+Google.Android.Material.Materialswitch
```

Comment:
Looks like this should be capitalized as `MaterialSwitch` to match the type name:

```
#### New Type: Google.Android.Material.Materialswitch.MaterialSwitch
```

Reply:
fixed